### DISCUSSION TITLE
Set `working-directory` differently between new tabs & splits

### DISCUSSION THREAD

**Discussion by Kristijan** [ghostty-org/ghostty#3636] [Feature Requests, Ideas] [answered]
I'd like the ability to control the behaviour of `working-directory` when creating a new tab or split.

This may be specific to my own workflow, but...

- When I'm working on a project, and need to create a new terminal, I'll create a split. In this instance, I want to inherit the working directory from one of the existing splits.
- When I'm working on something new, I'll create a new tab. In this instance, I want to start in my home directory.

For example, something along the lines of `working-directory-tab = home`,  `working-directory-split = inherit`, and fallback to `working-directory` if one is not explicity set.

**Comment by JakeWharton** [comment, marked as answer]
#1392?

**Comment by Kristijan** [comment]
> #1392?

Don't know why this didn't show up when I went searching 🤦‍♂️ 
Thanks @JakeWharton, I'll close this discussion.
